modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -499,7 +689,7 @@ const Fields = ({
 							name: 'type',
 						},
 						{
-							label: Liferay.Language.get('renderer'),
+							label: Liferay.Language.get('render-mode'),
```

Comment:
I'm not 100% convinced of this naming. We should harmonize it somehow as we are calling "FDS Cell Renderer" to the client extension that will end up contributing values to the dropdown. Something like "cell renderer" or "data renderer"?

Reply:
This is just for UI, so it can be anything. If we are already calling this a `Cell Renderer` in CE, makes sense to call it that here as well. As for FDS or no FDS, I'd say no FDS for now. I know that's a bit of can of worms, but Pablo seems to be inclined not to put FDS anywhere in UI.

Reply:
yep, no FDS sounds good to me too. My concern was more with the "Render Mode" vs "Cell Renderer"